### ISSUE TITLE
uno-check updates

### ISSUE THREAD

**Issue by spouliot** [unoplatform/uno.check#155]
Uno-check should check if the executing version is the latest version available. 

The first step could run `dotnet tool search uno.check` and compare the executing version with the latest one presently published.

```bash
$ dotnet tool search uno.check
Package ID      Latest Version      Authors          Downloads      Verified
----------------------------------------------------------------------------
uno.check       1.12.1              unoplatform      245070         x
```

This would simplify the documentation (one less step) and would be useful for people running it without reading the documentation (or that forgot that it might need to be updated).

An **auto update** would be even nicer but is more tricky since it would need to replace itself while running (which is tricky for some file systems).

**Comment by jeromelaban** [comment]
Martin added that in https://github.com/unoplatform/uno.check/pull/124, it should already have been published, though.

**Comment by spouliot** [comment]
Great! I filed this after updating an older version on a different computer. I'll keep an eye for the next update :)

**Comment by michael-hawker** [comment]
One note for this, is that this just broke our CI as by default it halts when a new version is available...

Though I feel like the line below was supposed to guard against this scenario??

https://github.com/unoplatform/uno.check/pull/124/files#diff-f9cea768413e43b41a1926674cce5457e08cf5be5e17318254f1143694f35cbaR315

We were running in CI mode though:

```
dotnet tool run uno-check --ci --fix --non-interactive --skip wsl --skip androidemulator --skip vswinworkloads --verbose
```

Seems like a bug then? I don't see any other way to suppress this behavior added in the linked PR...

I like the tool providing a warning still that there's an update, but Uno publishing an update to the tool shouldn't halt existing CI runs.

**Comment by jeromelaban** [comment]
@michael-hawker we had that issue as well, I've fixed it to 1.14.1, which won't cause issues in the future as long as there's the `--ci` flag. Apologies for the inconvenience!

**Comment by michael-hawker** [comment]
Thanks @jeromelaban upping to 1.14.1 and then adding the 1.13 manifest you suggested seems to have worked:

> --manifest https://raw.githubusercontent.com/unoplatform/uno.check/c3b289d7bb16a2a2df7f7f7f848d76fe1e74036d/manifests/uno.ui.manifest.json
